common/djangoapps/student/models.py

```diff
@@ -1382,6 +1386,22 @@ def refundable(self):
         else:
             return True
 
+    def _refund_window_end_date(self):
```

Comment:
I just realized why I've found this so confusing. What do you think of using the phrase "refund cutoff" instead of window? Window implies a start date as well. While the order placement date is the start date, it's not actually used.